.github/workflows/github-token-rotation-test.yml

```
@@ -0,0 +1,72 @@
+          
+          echo "Generating new GitHub token..."
+
+          # Authenticate as a GitHub App
+          jwt=$(python3 -c "import jwt, time; import os; print(jwt.encode({'iat': int(time.time()), 'exp': int(time.time()) + 600, 'iss': os.environ['GITHUB_APP_ID']}, os.environ['GITHUB_PRIVATE_KEY'].replace('\\n', '\n'), algorithm='RS256'))")
+
+          # Get installation ID
+          installation_id=$(curl -s \
+            -H "Authorization: Bearer $jwt" \
+            -H "Accept: application/vnd.github.v3+json" \
+            https://api.github.com/app/installations | jq -r '.[0].id')
+          
```

yamllint

[error] trailing spaces

(trailing-spaces)